app.js

```diff
@@ -1,26 +1,33 @@
 const express = require("express");
-const { getTopics } = require("./controllers/controller");
+const { getTopics, getArticleById } = require("./controllers/controller");
 
 const app = express();
 
 app.get('/api/topics', getTopics);
 
+app.get('/api/articles/:article_id', getArticleById);
+
 app.use(express.json());
 
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+});
+
 app.use("*", (req, res) => {
```

Comment:
I would move this app.use('*' ... )above the first error handling middleware (maybe to line 11) so it's below the valid endpoints but above the errors

Reply:
Fixed it. Thanks!